src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Avoid using literals in expressions (hardcoded values). Instead, use constants with descriptive names to explain what these values represent, as per the checklist item 2. For example, create constants for the minimum days required for a discount and the discount amounts themselves.